clippy.toml

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+disallowed-methods = [
```

Comment:
I prefer fix this instead of adding new clippy rules. Maybe we can call `tempfile.close()` by hand?

Reply:
I think it's difficult to change the behavior of `TempDir::into_path` because it's not a bug. It's a specification.
We can delete manually by `fs::remove_dir_all` (`tempfile.close` doesn't delete once `TempDir::into_path` is called) but my concern is the case someone can forget to delete by hand.
If someone don't know the behavior of `TempDir::into_path`, it can happen.

Reply:
Thank you for providing the information! I would like to refactor current test logic so that we don't need to create tempfile any more, thus we also don't need to add clippy rules here.

Reply:
O.K. I see.